### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by SebastianSvanland**
Setting up repo. Making a PR for review purposes.

For ROS projects using ISO22133, it would be nice to have ROS messages replicating the ISO22133 structs to follow the standard as much as possible when communicating between nodes. This would include the isoObject in repositories such as MondeoTestPlatform but also ATOS to allow ATOS to utilize ROS even more and split up the responsibility.

For example MondeoTestPlatform might not want all of atos_interfaces but rather only the parts specifically for iso22133. This repo would let us avoid duplicate definitions of messages.

### CODE REVIEW

**Comment by SebastianSvanland** [comment]
I'm not sure exactly how all of these should be defined so I'd be very grateful for any feedback

File: iso_msgs/msg/CoordinateSystem.msg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+uint32 COORDINATE_SYSTEM_ETRS89 = 0
+uint32 COORDINATE_SYSTEM_NAD83 = 1
+uint32 COORDINATE_SYSTEM_ITRF2000 = 2
+uint32 COORDINATE_SYSTEM_WGS84 = 3
+uint32 COORDINATE_SYSTEM_LOCAL = 4
+uint64 COORDINATE_SYSTEM_UNAVAILABLE = 255
```
Comment (victorjarlow) [inline iso_msgs/msg/CoordinateSystem.msg:10]: Why is this one uint64?

File: iso_msgs/msg/Rate.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+float64 monr;
```
Comment (victorjarlow) [inline iso_msgs/msg/Rate.msg]: Can the naming of this message be more clear? For example message rates. I assume that this message refers to the frequency at which monr, heab and monr2 are sent?

File: iso_msgs/msg/CoordinateSystem.msg (hunk 1 above)
Comment (SebastianSvanland) [reply iso_msgs/msg/CoordinateSystem.msg:10]: I don't remember actually. I did this particular message long ago in the Mondeo-repo but it should work with uint32. I'll test it and change

File: iso_msgs/msg/Rate.msg (hunk 2 above)
Comment (SebastianSvanland) [reply iso_msgs/msg/Rate.msg]: I'm trying to replicate the structure of the original [iso22133 structs](https://github.com/RI-SE/iso22133/blob/323bcd51a8f9da2c2a5a4b4eb7b3b7aa3a01ddc9/iso22133.h#L55) as much as possible, but that might not be the way to go. Maybe we need a meeting to clear up how we want to define and use these messages?

**Review by victorjarlow — APPROVED** [review]

**Comment by victorjarlow** [comment]
> I'm not sure exactly how all of these should be defined so I'd be very grateful for any feedback

A comment:
It would be very nice to have some feature in the future that generates these messages automatically each time the OSEM struct in the iso22133 repo is modified...

**Comment by SebastianSvanland** [comment]
@victorjarlow Why remove curvature if it's in iso22133? Even if it's not used, it's a 1-to-1 translation from the iso struct

**Comment by victorjarlow** [comment]
> @victorjarlow Why remove curvature if it's in iso22133? Even if it's not used, it's a 1-to-1 translation from the iso struct

This was the diff between the .zip file that you sent me and this repo. 
It's what made it work for me.

**Comment by victorjarlow** [comment]
> > @victorjarlow Why remove curvature if it's in iso22133? Even if it's not used, it's a 1-to-1 translation from the iso struct
> 
> This was the diff between the .zip file that you sent me and this repo. It's what made it work for me.

There might be some need for adaptation somewhere else, and keep curvature in the message in this repo

**Pull request merged**